raiden/tests/integration/api/rest/test_rest.py

```diff
@@ -557,138 +557,6 @@ def test_connect_insufficient_reserve(api_server_test_instance: APIServer, token
     assert "The account balance is below the estimated amount" in json_response["errors"]
 
 
-@raise_on_failure
-@pytest.mark.parametrize("number_of_nodes", [1])
-@pytest.mark.parametrize("channels_per_node", [0])
-@pytest.mark.parametrize("enable_rest_api", [True])
-def test_network_events(api_server_test_instance: APIServer, token_addresses):
-    # let's create a new channel
-    partner_address = "0x61C808D82A3Ac53231750daDc13c777b59310bD9"
-    token_address = token_addresses[0]
-    settle_timeout = 1650
-    channel_data_obj = {
-        "partner_address": partner_address,
-        "token_address": to_checksum_address(token_address),
-        "settle_timeout": str(settle_timeout),
-    }
-    request = grequests.put(
-        api_url_for(api_server_test_instance, "channelsresource"), json=channel_data_obj
-    )
-    response = request.send().response
-
-    assert_proper_response(response, status_code=HTTPStatus.CREATED)
-
-    request = grequests.get(
-        api_url_for(
-            api_server_test_instance,
-            "blockchaineventsnetworkresource",
-            from_block=GENESIS_BLOCK_NUMBER,
-        )
-    )
-    response = request.send().response
-    assert_proper_response(response, status_code=HTTPStatus.OK)
-    assert len(get_json_response(response)) > 0
-
-
-@raise_on_failure
-@pytest.mark.parametrize("number_of_nodes", [1])
-@pytest.mark.parametrize("channels_per_node", [0])
-@pytest.mark.parametrize("enable_rest_api", [True])
-def test_token_events(api_server_test_instance: APIServer, token_addresses):
-    # let's create a new channel
-    partner_address = "0x61C808D82A3Ac53231750daDc13c777b59310bD9"
-    token_address = token_addresses[0]
-    settle_timeout = 1650
-    channel_data_obj = {
-        "partner_address": partner_address,
-        "token_address": to_checksum_address(token_address),
-        "settle_timeout": str(settle_timeout),
-    }
-    request = grequests.put(
-        api_url_for(api_server_test_instance, "channelsresource"), json=channel_data_obj
-    )
-    response = request.send().response
-
-    assert_proper_response(response, status_code=HTTPStatus.CREATED)
-
-    request = grequests.get(
-        api_url_for(
-            api_server_test_instance,
-            "blockchaineventstokenresource",
-            token_address=token_address,
-            from_block=GENESIS_BLOCK_NUMBER,
-        )
-    )
-    response = request.send().response
-    assert_proper_response(response, status_code=HTTPStatus.OK)
-    assert len(get_json_response(response)) > 0
-
-
-@raise_on_failure
-@pytest.mark.parametrize("number_of_nodes", [1])
-@pytest.mark.parametrize("channels_per_node", [0])
-@pytest.mark.parametrize("enable_rest_api", [True])
-def test_channel_events(api_server_test_instance: APIServer, token_addresses):
```

Comment:
why is this removed?

Reply:
As @palango said:

> Removes /api/_debug/blockchain_events/*. Those used a full range RPC request which would most likely not work (on mainnet) and aren't actively used.

and this is the test for that part of the code, which should be removed along with the code itself.